ansible/roles/products/common/documentation/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+---
+- name: Ensure botocore and boto3 modules are installed
+  pip:
+    name: [ "boto3", "botocore"]
+    extra_args: "--user"
+  changed_when: false
+
+- name: Connect to Shadow AWS API
+  uri:
+    url: https://5vv2z6j3a7.execute-api.eu-west-2.amazonaws.com/prod
+    method: GET
+    return_content: yes
+    headers:
+      x-api-key: "{{ customer_key }}"
+  register: response
+  when: skip_molecule_task is not defined
+
+- name: Set temporary AWS credentials for accessing necessary files
+  set_fact:
+    aws_key: "{{ response.content | regex_search('(?<=ACCESS_KEY_ID=)(.*)(?=\\nSECRET_ACCESS_KEY)', multiline=True) }}"
+    aws_secret: "{{ response.content | regex_search('(?<=SECRET_ACCESS_KEY=)(.*)(?=\\nSESSION_TOKEN)', multiline=True) }}"
+    aws_security_token: "{{ response.content | regex_search('(?<=SESSION_TOKEN=)(.*)(?=\\nEXPIRATION)', multiline=True) }}"
+  when: skip_molecule_task is not defined
+
+- name: Set AWS credentials for accessing necessary files (for Molecule in AWS)
+  set_fact:
+    aws_key: "{{ lookup('env','AWS_ACCESS_KEY') }}"
+    aws_secret: "{{ lookup('env','AWS_SECRET_KEY') }}"
+  when: skip_molecule_task is defined
+
+- name: Choose documentation version
+  block:
+    - name: Choose Dexterous Hand Lite
+      set_fact:
+        documentation_folder: "documentation/dexterous_hand_lite"
+        documentation_file: "dexterous-hand-lite-test.pdf"
+        readthedocs_link: https://shadow-robot-company-dexterous-hand-lite.readthedocs-hosted.com/en/latest/
+        doc_icon_name: Dexterous Hand-Lite Documentation
+      when:
+        - "'arm_' not in product"
+        - "'glove_' not in product"
+        - "product == 'hand_lite' or product == 'hand_extra_lite'"
+        - product is defined
+    
+    - name: Choose Dexterous Hand
+      set_fact:
+        documentation_folder: "documentation/dexterous_hand"
+        documentation_file: "dexterous-hand-2.1.6.pdf"
+        readthedocs_link: https://shadow-robot-company-dexterous-hand.readthedocs-hosted.com/en/2.1.6/
```

Comment:
the other links are listed as variables, but here' its hardcoded. You probably mentioned this during S&T and I didn't catch the reason for hardcoding it here.